src/__tests__/components/__snapshots__/ExchangeQuoteComponent.test.tsx.snap

```diff
@@ -435,7 +435,7 @@ exports[`ExchangeQuote should render with loading props 1`] = `
             ]
           }
         >
-          0.00000001 BTC (&lt;0.01 EUR)
+          0 BTC (&lt;0.01 EUR)
```

Comment:
If you go into `src/util/fake/fakeRootState.ts` and add new `exchangeRatesMap` dummy data, this would get rid of these snapshot diffs.